aii-core/src/main/perl/aii-web.cgi

```diff
@@ -9,11 +9,14 @@
 # is missing the node will boot from the hard disk
 my $boothd = "default";
 
-# Quattor XML profiles
+# Quattor profiles
 my $web_root = "/var/www/html";
```

Comment:
I did it with Config::Simple but it adds a new dependency : perl-Config-Simple (see https://github.com/samary/aii/tree/read_cfg)

How do you suggest to do this in a cleaner way ?